src/Security/PostVoter.php

```diff
@@ -28,9 +28,9 @@ class PostVoter extends Voter
 {
     // Defining these constants is overkill for this simple application, but for real
     // applications, it's a recommended practice to avoid relying on "magic strings"
-    private const SHOW = 'show';
+    const DELETE = 'delete';
     private const EDIT = 'edit';
```

Comment:
it's ugly here that only the `EDIT` constant is left as private

solutions:
1) leave as is, ugly but correct
2) make constant public, even if not needed
3) make `EDIT` constant public, and use `{% if is_granted(constant('App\\Security\\PostVoter::EDIT'), post) %}` in https://github.com/symfony/demo/blob/master/templates/blog/post_show.html.twig#L62 instead of `{% if is_granted('edit', post) %}`

Reply:
I'd suggest (2) `make constant public, even if not needed` just for consistency.

Reply:
Yes, I also like the option (2).

Besides, I think we should add the `public` modifier explicitly to the constants to let developers know that we are making them public consciously. Thanks!

Reply:
visibility updated: now all constants are `public`